libr/main/radare2.c

```diff
@@ -487,6 +488,7 @@ R_API int r_main_radare2(int argc, const char **argv) {
 			break;
 		case 'B':
 			baddr = r_num_math (r->num, opt.arg);
+			baddr_set = true;
```

Comment:
just check if baddr == UT64_MAX insteaed of having baddr_set var

Reply:
are you sure?